.github/workflows/tests.yml

```
@@ -1,105 +1,23 @@
-          if-no-files-found: ignore
-
-      - name: Store static resources
-        uses: actions/upload-artifact@v4
-        with:
-          retention-days: 14
-          name: static-resources
-          path: |
-            database/nusa.sqlite
-            resources/static/**
-
   tests:
```

actionlint (1.7.7)

job "tests" needs job "configs" which does not exist in this workflow

(job-needs)